### PR TITLE
Added info about projects that use SymPy

### PR DESCRIPTION
**Pull request by ashutoshsaboo**
Hi @asmeurer @certik @moorepants ,

Resolves Issue #28 . I have added details about the projects that use SymPy. 

Please review the PR. Also, do advice, in case if any other details are required.

Thanks! :smile:

### CODE REVIEW

File: other_projects_that_use_sympy.tex [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  a symbolic computer algebra system (CAS) designed specifically for the
+  solution of problems encountered in field theory.
+\item
+  \href{https://mathics.github.io/}{\textbf{Mathics}}: Mathics is a
+  free, general-purpose online CAS featuring Mathematica-compatible
+  syntax and functions. It is backed by highly extensible Python code,
+  relying on SymPy for most mathematical tasks.
+\item
+  \href{http://mathpix.com/}{\textbf{Mathpix}}: An iOS App, that uses
+  Artifical Intelligence, to detect math handwritten input, and uses
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: math handwritten input --> handwritten math as input

File: other_projects_that_use_sympy.tex [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  \href{http://www.sagemath.org/}{\textbf{Sage}}: A CAS, visioned to be
+  a viable free open source alternative to Magma, Maple, Mathematica and
+  Matlab.
+\item
+  \href{https://cloud.sagemath.com}{\textbf{SageMathCloud}}:
+  SageMathCloud is a web-based cloud computing and course management
+  platform for computational mathematics.
+\item
+  \href{http://www.pydy.org/}{\textbf{PyDy}}: Multibody dynamics in
+  Python.
```
Comment (aktech) [inline other_projects_that_use_sympy.tex:41]: Multibody Dynamics with Python.

File: other_projects_that_use_sympy.tex (hunk 1 above)
Comment (aktech) [inline other_projects_that_use_sympy.tex]: I think comma is not required after Intelligence

**Comment by ashutoshsaboo** [comment]
@aktech Hi Thanks for your feedback. I have re-pushed the commit, with the required changes. Please provide feedback on the same. @asmeurer 

Thanks a lot! :smiley:

**Comment by Upabjojr** [comment]
https://github.com/jverzani/SymPy.jl

**Comment by isuruf** [comment]
http://octave.sourceforge.net/symbolic/

**Comment by ashutoshsaboo** [comment]
@isuruf @asmeurer @hargup @aktech @Upabjojr Does this seem fine now? I have also cleared the conflicts? :smiley:

File: other_projects_that_use_sympy.tex [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Such projects are listed below-:
+
+\begin{itemize}
+\item
+  \href{http://cadabra.science/index.html}{\textbf{Cadabra}}: Cadabra is
+  a symbolic computer algebra system (CAS) designed specifically for the
+  solution of problems encountered in field theory.
+\item
+  \href{http://octave.sourceforge.net/symbolic/}{\textbf{Octave-Forge}}:
```
Comment (isuruf) [inline other_projects_that_use_sympy.tex]: `Octave-Forge` -> `Octave Symbolic`

**Comment by aktech** [comment]
Tip: No need to write "Resolves Issue No 28" in the PR description, you should mention that in comments, see [this](https://help.github.com/articles/closing-issues-via-commit-messages/).

**Comment by ashutoshsaboo** [comment]
@aktech @isuruf Does this now seem fine and final? @hargup @asmeurer

File: other_projects_that_use_sympy.tex [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  \href{https://github.com/jverzani/SymPy.jl}{\textbf{SymPy.jl}}:
+  Provides a Julia interface to SymPy using PyCall.
+\item
+  \href{https://mathics.github.io/}{\textbf{Mathics}}: Mathics is a
+  free, general-purpose online CAS featuring Mathematica-compatible
+  syntax and functions. It is backed by highly extensible Python code,
+  relying on SymPy for most mathematical tasks.
+\item
+  \href{http://mathpix.com/}{\textbf{Mathpix}}: An iOS App, that uses
+  Artifical Intelligence to detect handwritten math as input, and uses
```
Comment (isuruf) [inline other_projects_that_use_sympy.tex]: `Artificial`

**Comment by ashutoshsaboo** [comment]
Ready for merge? @aktech @hargup @asmeurer Or any more changes? @isuruf @Upabjojr

File: other_projects_that_use_sympy.tex [hunk 5]
```diff
@@ -0,0 +1,69 @@
+There are several projects that use SymPy as a library for implementing
+a part of their project, or even as a part of back-end for their app as
+well.
+
+Such projects are listed below-:
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: Such --> Some of them

File: other_projects_that_use_sympy.tex [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+\item
+  \href{http://www.sagemath.org/}{\textbf{Sage}}: A CAS, visioned to be
+  a viable free open source alternative to Magma, Maple, Mathematica and
+  Matlab.
+\item
+  \href{https://cloud.sagemath.com}{\textbf{SageMathCloud}}:
+  SageMathCloud is a web-based cloud computing and course management
+  platform for computational mathematics.
+\item
+  \href{http://www.pydy.org/}{\textbf{PyDy}}: Multibody dynamics with
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: dynamics --> Dynamics

**Comment by ashutoshsaboo** [comment]
@aktech I have re-pushed the commit with the required changes. I would suggest you to merge this PR - #31 first, and then I'll fix conflicts in #29 , since both of them will have conflicts then. :smiley: 

So, ready for merge?

File: other_projects_that_use_sympy.tex [hunk 7]
```diff
@@ -0,0 +1,69 @@
+There are several projects that use SymPy as a library for implementing
+a part of their project, or even as a part of back-end for their app as
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: `app` doesn't seems apt for a paper, I would suggest to write `application`.

**Comment by ashutoshsaboo** [comment]
@aktech Yes :+1: . Corrected above^

File: other_projects_that_use_sympy.tex [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+well.
+
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: This empty line doesn't leaves a blank line, see [this](https://www.sharelatex.com/learn/Line_breaks_and_blank_spaces#Line_breaks).

**Comment by ashutoshsaboo** [comment]
@aktech If this seems fine, then we may merge this first before #29 before #31  as well? That'd also be fine with me. :smiley: Also, this has passed all the tests as well.

File: other_projects_that_use_sympy.tex [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+  \href{http://octave.sourceforge.net/symbolic/}{\textbf{Octave Symbolic}}:
+  The Octave-Forge Symbolic package adds symbolic calculation features
+  to GNU Octave. These include common Computer Algebra System tools such
+  as algebraic operations, calculus, equation solving, Fourier and
+  Laplace transforms, variable precision arithmetic and other features.
+\item
+  \href{https://github.com/jverzani/SymPy.jl}{\textbf{SymPy.jl}}:
+  Provides a Julia interface to SymPy using PyCall.
+\item
+  \href{https://mathics.github.io/}{\textbf{Mathics}}: Mathics is a
```
Comment (aktech) [inline other_projects_that_use_sympy.tex:22]: Latex warning here:
Overfull \hbox (25.15338pt too wide) in paragraph at lines 22--27

**Comment by ashutoshsaboo** [comment]
@aktech Ohh yeah I see that appears a couple of times in the output log ( I didn't see it that closely earlier). How do I solve that? But I guess, that doesn't give any problem, even if we don't solve that issue since it's only a warning and not an error. @aktech ?

**Comment by aktech** [comment]
> How do I solve that?

One way is to remove hyphen between Mathematica and compatible, if it makes sense and another way is to use `\\` for new line before Mathematica.

**Comment by ashutoshsaboo** [comment]
@aktech Yeah! So, now it doesn't show the LaTeX warning - `Overfull \hbox (25.15338pt too wide) in paragraph at lines 22--27` as well. Any other changes that you'd advice of?

**Comment by asmeurer** [comment]
Don't manually add newlines. Generally you can fix these things by defining hyphenation for a word (see https://en.wikibooks.org/wiki/LaTeX/Text_Formatting#Hyphenation). Probably LaTeX doesn't know how to hyphenate "Math-e-ma-ti-ca".

**Pull request merged**